application/listeners/posthelpers.go

```diff
@@ -9,15 +9,10 @@ import (
 	"github.com/silinternational/wecarry-api/notifications"
 )
 
-// This is intended as a temporary fill-in
-func getUserLanguage() string {
-	return "en"
-}
-
 var argAppName = map[string]string{"AppName": domain.Env.AppName}
 
-func getTranslatedSubject(translationID, template string) string {
-	subj, err := domain.TranslateWithLang(getUserLanguage(), translationID, argAppName)
+func getTranslatedSubject(language, translationID, template string) string {
+	subj, err := domain.TranslateWithLang(language, translationID, argAppName)
 
 	if err != nil {
 		domain.ErrLogger.Printf("error translating '%s' notification subject, %s", template, err)
```

Comment:
Just a thought: what if you logged the subject itself, and not pass in the template?

Reply:
Do you mean, log the `translationID`?

Reply:
Yes